tests/ft_sh.vader

```diff
@@ -1,14 +1,58 @@
 Include: include/setup.vader
 
 Execute (shellcheck):
-  AssertEqual neomake#makers#ft#sh#shellcheck()['args'], ['-fgcc']
-  set ft=sh
-  AssertEqual neomake#makers#ft#sh#shellcheck()['args'], ['-fgcc', '-s', 'sh']
-  set ft=bash
-  AssertEqual neomake#makers#ft#sh#shellcheck()['args'], ['-fgcc', '-s', 'bash']
-  set ft=nadda
-  AssertEqual neomake#makers#ft#sh#shellcheck()['args'], ['-fgcc']
+  " By variable...
+  let g:is_sh = 1
+  AssertEqual ['-fgcc', '-s', 'sh'], neomake#makers#ft#sh#shellcheck().args
+  unlet g:is_sh
```

Comment:
What would be the net gain here? As far as I see `Save`/`Restore` are used for variables only. (Vader's documentation could be better. :-P)